keystrokes/src/components/Character/CharacterCreationModal.js

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from 'react';
+import {
+	Modal,
+	Button,
+	TextField,
+	FormControl,
+	InputLabel,
+	Select,
+	MenuItem,
+	Grid,
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { characterDetails } from '../../config/constants';
+import styleGuide from '../../stylesheets/style';
+
+CharacterCreationModal.propTypes = {
+	handleSubmit: PropTypes.func.isRequired,
+	isOpen: PropTypes.bool.isRequired,
+	setIsOpen: PropTypes.bool.isRequired,
+};
+
+function canSave() {
```

Comment:
these variables don't live outside of the scope of the CharacterCrreationModal so you'll probably want to define it in there.